app/views/shared/_edit_form.html.erb

```diff
@@ -47,14 +47,14 @@
 
 <% if current_organization.show_driving_reimbursement && resource.role == "Volunteer" %>
   <div class="input-style-1">
-    <%= f.label :address_attributes_content, "Mailing address" %>
+    <%= f.label :address_attributes_content, "Address" %>
     <% if policy(resource).update_user_setting? %>
       <%= f.fields_for :address, (resource.address ? nil : Address.new) do |a| %>
-        <%= a.text_field :content, placeholder: "Mailing Address" %>
```

Comment:
can you add the placeholders back?

Reply:
I removed those because in the volunteer view of the their own profile edit page, we are not showing any placeholders for this field. So did so to match the visual representation of both views. :sweat_smile: 
Anyway, I've added back the placeholders with their updated value.